docs/manual.md

```diff
@@ -668,6 +668,18 @@ For simulating the ZCU102 using QEMU, use the following command:
        -device loader,addr=0xfd1a0104,data=0x0000000e,data-len=4 \
        -serial mon:stdio
 
+**Note**:
+
+The Xilinx U-Boot fork `u-boot-xlnx` does not honor the [`ARMV8_SWITCH_TO_EL1` configuration setting](https://github.com/Xilinx/u-boot-xlnx/blob/684585acb001e161320720b349ad572aedfc24bd/arch/arm/cpu/armv8/Kconfig#L38) on the zynqmp platform (which the ZCU102 is part of).
+Instead, a `go 0x40000000` in U-Boot is [unconditionally always switching to EL1](https://github.com/Xilinx/u-boot-xlnx/blob/684585acb001e161320720b349ad572aedfc24bd/board/xilinx/zynqmp/zynqmp.c#L233
+).
+Being a hypervisor, Microkit has to run on EL2, so when booting with Xilinx' U-Boot fork, the seL4 loader will detect the wrong EL and subsequently abort the boot.
```

Comment:
"Being a hypervisor" this is not currently true on mainline Microkit.

I will review this once I add virtualisation support to Microkit (hopefully within the next week).

Reply:
Took longer than I expected but I have merged virtualisation support now.

Reply:
```suggestion
The ZCU102 is configured to run in hypervisor mode which means that the Microkit system image must run at EL2 or EL3. When booting with U-Boot, the Microkit boot-loader will detect the wrong EL and subsequently abort the boot.
```